README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider adding a brief description before each command to explain what it does. This can help users who may not be familiar with the commands understand their purpose.